### ISSUE TITLE
Issue when selecting dates within a 10 day GIOPS 

### ISSUE THREAD

**Issue by dwayne-hart** [DFO-Ocean-Navigator/Ocean-Data-Map-Project#653]
**Describe the bug**
When looking at the 10 Day Forecast for either LatLon or Polar Stereographic. We are able to see the 10th date's forecast but not any of the days prior.

**Comment by dwayne-hart** [comment]
> [2020-02-16 22:44:51 -0400] [3328] [ERROR] Error handling request /api/v1.0/tiles/gaussian/25/10/EPSG:3857/giops_fc_10day_2dps/votemper/2213827200/0/-5,30/0/0/0.png
> Traceback (most recent call last):
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/xarray/core/ops.py", line 187, in _call_possibly_missing_method
>     method = getattr(arg, name)
> AttributeError: 'Array' object has no attribute 'argsort'
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/gunicorn/workers/sync.py", line 135, in handle
>     self.handle_request(listener, req, client, addr)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
>     respiter = self.wsgi(environ, resp.start_response)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1997, in __call__
>     return self.wsgi_app(environ, start_response)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1985, in wsgi_app
>     response = self.handle_exception(e)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1540, in handle_exception
>     reraise(exc_type, exc_value, tb)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/_compat.py", line 33, in reraise
>     raise value
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1982, in wsgi_app
>     response = self.full_dispatch_request()
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1614, in full_dispatch_request
>     rv = self.handle_user_exception(e)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1517, in handle_user_exception
>     reraise(exc_type, exc_value, tb)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/_compat.py", line 33, in reraise
>     raise value
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1612, in full_dispatch_request
>     rv = self.dispatch_request()
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1598, in dispatch_request
>     return self.view_functions[rule.endpoint](**req.view_args)
>   File "/home/buildadm/ocean-nav/Ocean-Data-Map-Project/routes/api_v1_0.py", line 410, in tile_v1_0
>     return routes.routes_impl.tile_impl(projection, interp, radius, neighbours, dataset, variable, time, depth, scale, zoom, x, y)
>   File "/home/buildadm/ocean-nav/Ocean-Data-Map-Project/routes/routes_impl.py", line 567, in tile_impl
>     'scale': scale,
>   File "/home/buildadm/ocean-nav/Ocean-Data-Map-Project/plotting/tile.py", line 176, in plot
>     with open_dataset(config, variable=variable, timestamp=time) as dataset:
>   File "/home/buildadm/ocean-nav/Ocean-Data-Map-Project/data/mercator.py", line 37, in __enter__
>     self.__latsort = np.argsort(self.latvar[:])
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 1034, in argsort
>     return _wrapfunc(a, 'argsort', axis=axis, kind=kind, order=order)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 56, in _wrapfunc
>     return getattr(obj, method)(*args, **kwds)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/xarray/core/variable.py", line 1660, in func
>     return self.__array_wrap__(f(self.data, *args, **kwargs))
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/xarray/core/ops.py", line 207, in func
>     return _call_possibly_missing_method(self, name, args, kwargs)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/xarray/core/ops.py", line 189, in _call_possibly_missing_method
>     duck_array_ops.fail_on_dask_array_input(arg, func_name=name)
>   File "/home/sdfo000/ocean-nav/miniconda/3/envs/navigator/lib/python3.6/site-packages/xarray/core/duck_array_ops.py", line 59, in fail_on_dask_array_input
>     raise NotImplementedError(msg % func_name)
> NotImplementedError: 'argsort' is not yet a valid method on dask arrays

**Comment by htmlboss** [comment]
We're not using the `self.__latsort` or `self.__lonsort` attributes so they can be deleted to resolve this issue.

**Comment by douglatornell** [comment]
That's the exception I was talking about a couple of Tuesdays ago, but I wasn't convinced that it wasn't an artefact of my incomplete refactoring. I was seeing it when I tried to use the new `Mercator` class to handle polar stereographic datasets. They have model x-y dimensions and `nav_lon, nav_lat` variables, and that triggers the exception. It went away for me when I implemented the new `Nemo` class, and used that to handle polar stereographic datasets.
